guide/ops/high-availability/high-availability-supplemental.md

```diff
@@ -0,0 +1,142 @@
+---
+title: High Availability (Supplemental)
+layout: website-normal
+---
+
+This document supplements the High Availability documentation available [here](http://brooklyn.apache.org/v/latest/ops/high-availability.html)
```

Comment:
Use `This document supplements the [High Availability]]({{ site.path.guide }}/ops/high-availability.html) documentation.` Don't use absolute links to things in the docs; and I have a personal preference for the link not being "here", but being the name of what it's linking to.